charts/mirrorbits-parent/templates/_helpers.tpl

```diff
@@ -1,6 +1,7 @@
 {{/* vim: set filetype=mustache: */}}
 {{/*
-Expand the name of the chart.
+Define the full "release" (e.g. chart installation) name.
+Must be 63 chars. maximum.
```

Comment:
```suggestion
```
Not sure I would add this.

Reply:
Added it because it is not an implementation detail, but a hard functional requirement (hence having it in the template function pre-comment).